build/deploy/Dockerfile

```diff
@@ -3,9 +3,9 @@ FROM debian:jessie
 MAINTAINER Tobias Schottdorf <tobias.schottdorf@gmail.com>
 
 RUN mkdir -p /cockroach
-COPY cockroach.sh test.sh build/cockroach /cockroach/
 COPY build/resources /cockroach/resources
 COPY build/ui /cockroach/ui
+COPY cockroach.sh test.sh build/cockroach /cockroach/
```

Comment:
no, the above will actually copy the **contents** of `build/resources` into `/cockroach`, which isn't what we want.
I would have used `COPY build /cockroach` but the `build` directory holds also the various test binaries which we don't want in there.

I think we should embed the contents of those two directories in the binary anyways.